frontend/src/pages/Dashboard.vue

```diff
@@ -43,6 +50,12 @@
             ...mapActions('dashboard', {
                 fetchingActivityChartData: FETCHING_ACTIVITY_CHART_DATA,
             }),
-        }
+        },
+        data () {
```

Comment:
it seems you not using local state, so it can be removed

Reply:
it was for some other component, didn't not notice it was not being used